README.md

```diff
@@ -1,7 +1,140 @@
-# nim-codex
+# Codex Decentralized Durability Engine
+
+> The Codex project aims to create a decentralized durability engine that allows persisting data in p2p networks. In other words, it allows storing files and data with predictable durability guarantees for later retrieval.
+
+> WARNING: This project is under active development and is considered pre-alfa.
 
 [![License: Apache](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 [![License: MIT](https://img.shields.io/badge/License-MIT-blue.svg)](https://opensource.org/licenses/MIT)
 [![Stability: experimental](https://img.shields.io/badge/stability-experimental-orange.svg)](#stability)
 [![CI](https://github.com/status-im/nim-codex/actions/workflows/ci.yml/badge.svg?branch=main)](https://github.com/status-im/nim-codex/actions?query=workflow%3ACI+branch%3Amain)
 [![Codecov](https://codecov.io/gh/status-im/nim-codex/branch/main/graph/badge.svg?token=XFmCyPSNzW)](https://codecov.io/gh/status-im/nim-codex)
+[![Discord](https://img.shields.io/discord/895609329053474826)](https://discord.gg/CaJTh24ddQ)
+
+
+## Build and Run
+
+To build the project, clone it and run:
+
+```bash
+make update && make exec
```

Comment:
Just curious, where does the `exec` target come from?

It works, but when I checked nim-codex's Makefile and the [`.mk` files](https://github.com/status-im/nimbus-build-system/tree/master/makefiles) of nimbus-build system it wasn't clear where `exec` is coming from.